terminus/geometry/arc.py

```diff
@@ -76,6 +91,12 @@ def length(self):
         """
         return abs(math.pi * self._radius * self._angular_length / 180.0)
 
+    def counter_clockwise(self):
```

Comment:
Can you add a comment here explaining what this does? Also, we generally prefer to rule cases early and then leave the core implementation, like:
```
if self.angular_length() >= 0:
    # Nothing to do, already counter clockwise
    return self
return Arc(self.end_point(), self.end_heading() + 180, self.radius(), -self.angular_length())
```